A67-xds-dynamic-parameters.md

```diff
@@ -0,0 +1,246 @@
+## xDS Dynamic Parameters
+
+*   Author(s): @allanrbo
+*   Approver: @markdroth
+*   Status: Draft
+*   Implemented in:
+*   Last updated: 2023-07-21
+*   Discussion at:
```

Comment:
Please go ahead and create the mailing list thread, and then add a link to it here, as per the gRFC process.

Reply:
Still need to take care of this.